spec/controllers/idv/socure/document_capture_controller_spec.rb

```diff
@@ -183,7 +194,17 @@
     it 'returns OK (200)' do
       post(:update)
 
-      expect(response).to have_http_status(:ok)
+      expect(response).to redirect_to(idv_ssn_path)
+    end
+
+    context 'when doc auth fails' do
+      let(:doc_auth_success) { false }
+
+      it 'redirects to document capture' do
+        post(:update)
```

Comment:
should all the update request get changed to GET?
```suggestion
        get(:update)
```